keps/sig-auth/5018-dra-adminaccess/README.md

```diff
@@ -466,7 +466,7 @@ ResourceClaimTemplate and ResourceClaim for admin access
 
 - Gather feedback
```

Comment:
Missing bits higher in the doc:
1. make sure to check appropriate boxes in `Release Singoff Checklist`
2. In `Integration tests` section, please make sure to link tests according [to the template](https://github.com/kubernetes/enhancements/blob/master/keps/NNNN-kep-template/README.md#integration-tests), especially the newly added that are called out there, since looking at the PRs submitted during alpha they did add new tests.

Reply:
These comments still hold.